demes/demes.py

```diff
@@ -57,8 +57,10 @@ def validator(self, attribute, value):
 class Epoch:
     """
     Population size parameters for a deme in a specified time period.
-    Times follow the backwards-in-time convention (time values increase
-    from the present towards the past).
+    Times follow the forwards-in-time convention (time values increase
+    from the present towards the past). The start time of the epoch is
+    the more ancient time, and the end time is more recent, so that the
+    start time must be greater than the end time
```

Comment:
That is a bit confusing, but I wanted to avoid having to write negative numbers. Not sure what the best solution for anchoring and measuring relative time is.